vnext/Microsoft.ReactNative/Modules/DeviceInfoModule.cpp

```diff
@@ -30,14 +32,17 @@ void DeviceInfoHolder::InitDeviceInfoHolder(
     propertyBag.Set(DeviceInfoHolderPropertyId(), std::move(deviceInfoHolder));
 
     auto const &displayInfo = winrt::Windows::Graphics::Display::DisplayInformation::GetForCurrentView();
-    auto const &window = xaml::Window::Current().CoreWindow();
-
-    deviceInfoHolder->m_sizeChangedRevoker =
-        window.SizeChanged(winrt::auto_revoke, [weakHolder = std::weak_ptr(deviceInfoHolder)](auto &&, auto &&) {
-          if (auto strongHolder = weakHolder.lock()) {
-            strongHolder->updateDeviceInfo();
-          }
-        });
+    /// TODO: WinUI 3 islands support
```

Comment:
>/// [](start = 4, length = 3)

Please use double slash for normal comments. The triple slash line comments are usually used for automated docs.